lib/game.rb

```diff
@@ -0,0 +1,79 @@
+class Game
+
+attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+[0, 1, 2], [3, 4, 5],
+[6, 7, 8], [0, 4, 8],
+[2, 4, 6], [0, 3, 6],
+[1, 4, 7], [2, 5, 8],
+]
+
+def initialize(player_1 = Players::Human.new("X") , player_2 = Players::Human.new("O"), board = Board.new)
+  @player_1 = player_1
+  @player_2 = player_2
+  @board = board
+end
+
+def current_player
+ return @player_1 if board.turn_count.even?
```

Comment:
This might be a good place to use a ternary operator. `@board.turn_count.even? ? @player_1 : @player_2` Avoid the return keyword!